.github/actions/local-lambda/action.yml

```diff
@@ -1,23 +1,38 @@
 name: 'Docker Build and Run'
 
 inputs:
-  folder:
+  path:
     description: 'folder for the lambda code'
     required: true
-  command:
+  handler:
     description: 'lambda handler'
     required: true
+  github_token:
+     description: 'github token'
+     required: false
 
 runs:
   using: composite
   steps:
     - name: 'Checkout repository'
       uses: actions/checkout@v2
 
+    - name: Login
```

Comment:
this uses the github docker registry (something organizations have) to cache docker layers. It's the most efficient solution I found, and since performance is not really an issue that's everything I will do for now.